File: 08-maratona/superior/c_ordenado/c_ordenado-2.py
num = int(input())
valores = input().split(" ")[:num]

result = True

for i in range(len(valores)-1):
    if int(valores[i]) > int(valores[i+1]):
        result = False
        break


if result:
    print("SIM")
else:
    print("NAO")

File: 2018-10-VIII-Maratona-POP/superior/c_ordenado/c_ordenado-2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int v[300];
  int n;

  cin>>n;

  for(int i=0;i<n;i++){
    cin>>v[i];
  }
  for(int i=0;i<n-1;i++){
    if(v[i+1] < v[i]){
      cout<<"NAO\n";
      return 0;
    }
  }

  cout<<"SIM\n";
  return 0;
}

